policy.go

```diff
@@ -103,6 +110,14 @@ type PolicyCreateOptions struct {
 	// Required: The name of the policy.
 	Name *string `jsonapi:"attr,name"`
 
+	// **Note: This field is still in BETA and subject to change.**
+	// Optional: The underlying technology that the policy supports.
```

Comment:
might be helpful if comment includes that this field defaults to `sentinel` if not specified for PolicyCreate.